modules/nlu/src/views/full/entities/CreateEntityModal.jsx

```diff
@@ -36,7 +35,7 @@ export default class CreateEntityModal extends React.Component {
 
   createEntity = e => {
     const entity = {
-      id: sanitizeFilenameNoExt(this.state.name),
+      id: this.state.name.toLowerCase().replace(/\t\s/),
```

Comment:
This only lowercases the name to form the ID. Why do we remove replacement of spaces/tab with "-" ? Isn't .replace without a second argument useless?

Reply:
I simply copy pasted the sanitizeFilnameNoExt func

Reply:
But it still seems to be missing a part (and replace all occurrences)

![image](https://user-images.githubusercontent.com/42552874/67210262-c7b3fc80-f3e6-11e9-9cd9-d5383869472c.png)